docs/README.md

```diff
@@ -121,7 +121,7 @@ You will be able to select the platform you want to create the component.
 ? Component Name: ExampleComponent
 ? Generate for:
 > Web
-> React native
+> React Native
```

Comment:
since it's the name of the product we should treat it as a proper noun.

I hope my English teacher is proud of me.